pkg/k8sutil/k8sutil.go

```diff
@@ -0,0 +1,82 @@
+package k8sutil
```

Comment:
This package used by `pkg/metrics/service.go` only. To get the operator name and namespace.
And those to functions can be replaced by const `config.OperatorName` and `config.OperatorNamespace`

So that, we do not need to introduce this package at all.

Reply:
Ah. I added this to import the old code that was removed from the controller.

Thanks for the suggestion.
I would be happy to switch to a different solution.
I will see if I can work out how to do that.

Reply:
I can basically replace the variable setup in service.go with this correct?
```
	operatorName := config.OperatorName
	namespace := config.OperatorNamespace
```

Reply:
Yeah, I think it would work the same. The current `GetOperatorNamespace()` or `GetOperatorName()` just return the same value as `config.OperatorNamespace` and `config.OperatorName`

Reply:
I have made that change locally and the tests are passing.
I will push that up and the rest of my changes.

Thanks for the suggestion. :)